src/opencl_DES_bs_plug.c

```diff
@@ -1145,7 +1145,7 @@ size_t create_keys_kernel_set_args(int mask_mode)
 	cl_ulong const_cache_size;
 	int i;
 
-	if (mask_mode) {
+	if (mask_mode && mask_int_cand_target) {
```

Comment:
self-reviewing this, it's not sufficient. I need to change the callers and ultimately whatever sets `mask_mode`.

Reply:
I will remove the LM and DEScrypt bugfixes from this PR and open an issue.